portal-frontend/src/app/shared/button-container/button-container.component.scss

```diff
@@ -0,0 +1,49 @@
+@use '../../../styles/functions' as *;
+@use '../../../styles/colors';
+
+
+  .button-container {
```

Comment:
Nitpick: probably shouldn't be indented. Auto-format on save should have caught this.